modules/backend/calendar/WorkUnit.hpp

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include <memory>
+#include <vector>
+
+#include "../clinic/room/iRoom.hpp"
+#include "../clinic/staff/doctors/iDoctor.hpp"
```

Comment:
extend target_link_libraries in CmakeLists with `clinic-lib` and you should be able to use includes as follow:
`room/iRoom.hpp` `doctors/iDoctor.hpp`

Reply:
done